### ISSUE TITLE
[Bug]: No highlight for inline command text in dark mode of readthedocs pages

### ISSUE THREAD

**Issue by migueldivo** [canonical/vanilla-framework#5459]
**Describe the bug**

Currently within readthedocs documentation there is no highlight colour for commands which are inline in a paragraph/sentence in dark mode.

In light mode, highlight colour for inline commands is present. 
![Image](https://github.com/user-attachments/assets/3e3d3154-8333-4ac6-ba4d-4479c9fea773)

In dark mode, there isn't any highlighting present.
![Image](https://github.com/user-attachments/assets/3468c3cd-0529-4aa0-8988-832ad574519f)

**To Reproduce**

1. Go to https://canonical-charmcraft.readthedocs-hosted.com/en/latest/tutorial/write-your-first-kubernetes-charm-for-a-django-app/#write-your-first-kubernetes-charm-for-a-django-app
2. Change to dark mode
3. refer to light mode version to spot highlighted inline commands
4. See that in dark mode those same inline commands are not highlighted

**Expected behavior**

When switching to dark mode, the same inline commands that are highlighted in light mode should be highlighted in dark mode as well.

CSS command for dark mode: `color-inline-code-background: rgba(255, 255, 255, .03);`

**Screenshots**

A global CSS command is present to set the colour of the inline command highlight to `rgba(0, 0, 0, .03)`
![Image](https://github.com/user-attachments/assets/b2c66117-d3d0-4e72-863b-3bd9b1d8f472)

Within the dark mode class, there isn't a CSS command, therefore, it uses the global CSS command which is a tinted black in dark mode.
![Image](https://github.com/user-attachments/assets/f64d88e4-8fba-4d03-ad34-7425bf4dcff0)

**Desktop (please complete the following information):**

- OS: Ubuntu 24.04
- Browser Firefox
- Version Latest/stable 134.0.2-1

**Comment by jmuzina** [comment]
Hi @migueldivo , thanks for the report!

I copied some markup from the docs page you linked into a [codepen](https://codepen.io/jmuzina/pen/KwPrqvP) and was able to get themed highlighting for command text in all themes using the current version of Vanilla. 

<img width="630" alt="Image" src="https://github.com/user-attachments/assets/25fac53e-33ae-42a5-9ca0-9d85fcdf9f4e" />

Could you describe how you're configuring Vanilla for this project and what version you are using? I inspected the command element and found that it is not using Vanilla 4.20.2 styling, rather it seems to be using styling from [Furo](https://github.com/canonical/canonical-sphinx/blob/f2bf311c10cde0b1938f5d0173d9d88132e9a422/canonical_sphinx/theme/static/furo_colors.css#L16)

<img width="472" alt="Image" src="https://github.com/user-attachments/assets/961fb103-6a51-484f-84ff-fd6616d3e25f" />

From what I understand of the code you have shown me, you'd need to add a new color value for `color-inline-code-background` in the dark theme. That variable isn't provided by Vanilla, though indeed many [Canonical repos are using it](https://github.com/search?q=org%3Acanonical+color-inline-code-background&type=code&p=2). I think these come from [canonical/canonical-sphinx](https://github.com/canonical/canonical-sphinx).

Let me know if I've understood the problem incorrectly, this is my first time directly supporting a documentation sites issue :)

**Comment by migueldivo** [comment]
Hi @jmuzina thanks for the reply! I think you are right, this would need to be an issue within [canonical/canonical-sphinx](https://github.com/canonical/canonical-sphinx). I'm fairly new to Canonical and this is the first time opening an issue so I wasn't aware there was a Sphinx specific repo. 

I will add this issue to the Sphinx repo and as well [this other issue](https://github.com/canonical/vanilla-framework/issues/5460) I made that should also belongs in Sphinx. 

Thank you again for bringing it up!

**Comment by migueldivo** [comment]
This issue has been added to [canonical/canonical-sphinx](https://github.com/canonical/canonical-sphinx/issues/65)

https://github.com/canonical/canonical-sphinx/issues/65

**Comment by jmuzina** [comment]
Glad I could help @migueldivo , welcome to Canonical and let us know if we can be of help :)
